src/utils/validation.js

```diff
@@ -145,6 +145,18 @@ export const requireFields = (...fields) => (req, res, next) => {
   return next(!isEmpty(errors) && new ValidationError(errors));
 };
 
+export const requireFieldsAny = (...fields) => (req, res, next) => {
+  const errors = {};
+
+  if (!Object.keys(req.body).some(field => fields.includes(field))) {
+    fields.forEach(field => {
+      errors[field] = 'errors.fieldListOneOfRequired';
+    });
+  }
+
+  return next(!isEmpty(errors) && new ValidationError(errors));
```

Comment:
`next(false)` - not interpreted as an error?

Reply:
I took this from `requireFields()` function so I guess we are ok with that. Also I've tried this API: works fine on valid requests.